src/ast.js

```diff
@@ -219,6 +219,46 @@ export class FunctionDefinition extends ASTNode {
   }
 }
 
+export class CondClause extends ASTNode {
+  constructor(from, to, testExpr, thenExprs, options={}) {
+    super(from, to, 'condClause', options);
+    this.testExpr = testExpr;
+    this.thenExprs = thenExprs;
```

Comment:
Why isn't this `thenExpr`? Is this for language-agnosticism, to allow for a sequence of `then` statements?